src/contexts/global.tsx

```diff
@@ -26,7 +26,7 @@ import { OnOff } from '@/types/generics'
 import { NetworkType, networkTypes } from '@/types/network'
 import { SnackbarMessage } from '@/types/ui'
 
-interface GlobalContextInterface {
+export interface GlobalContextInterface {
```

Comment:
nit: In our other apps we call these interfaces `[name]Props`. Maybe you'd like to change it now that you are exporting it?

Reply:
Mhh good question! I associate props to components ("React props"). It kinda makes sense for me to use another naming scheme here.

Reply:
Indeed, this TS interface does not represent props, but the **value** of the context. So, how about `GlobalContextValue`?

Reply:
Why not!